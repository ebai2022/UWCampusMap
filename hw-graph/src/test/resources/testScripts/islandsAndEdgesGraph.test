# Test graph with both islands and edges

# Create a graph
CreateGraph A

# Add nodes
AddNode A e1
AddNode A e2
AddNode A e3

# Add edge
AddEdge A e2 e3 connected

# Check children
ListChildren A e1
ListChildren A e2
ListChildren A e3