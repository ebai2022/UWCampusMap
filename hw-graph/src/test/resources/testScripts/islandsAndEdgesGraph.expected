# Test graph with both islands and edges

# Create a graph
created graph A

# Add nodes
added node e1 to A
added node e2 to A
added node e3 to A

# Add edge
added edge connected from e2 to e3 in A

# Check children
the children of e1 in A are:
the children of e2 in A are: e3(connected)
the children of e3 in A are: